features/admin.core.v1/components/header.tsx

```diff
@@ -198,6 +206,54 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (props: HeaderPro
                     transformOrigin={ { horizontal: "right", vertical: "top" } }
                     onClose={ onCloseHelpMenu }
                 >
+                    { window["AppUtils"].getConfig().extensions.helpCenterUrl && (
```

Comment:
Will it be possible to get this from redux state?

Reply:
This code was not changed, hence discussed to fix this later.